### PR TITLE
Div Game C++ Solution

### PR DESCRIPTION
**Pull request by maggie-j-liu**
I added a C++ solution to AC Div Game. I linked the [editorial for the problem](https://atcoder.jp/contests/abc169/tasks/abc169_d/editorial), but there is no editorial there yet. Should I link the [overall editorial](https://img.atcoder.jp/abc169/editorial.pdf#page=12) instead?

### CODE REVIEW

**Comment by dolphingarlic** [comment]
Could you just link to the overall editorial (with the page number in the URL) in the problem link? Since there's already an editorial and you can view other people's submissions on AC, I don't think a whole new file is necessary

**Comment by thecodingwizard** [comment]
~~additional solutions don't hurt right?~~ actually idrk :P

**Comment by maggie-j-liu** [comment]
Isn't it better to have the solution and the explanation in the same place so it's more convenient? If not, I can just close this.

**Comment by maggie-j-liu** [comment]
There is an English explanation if you scroll down a few pages.

**Comment by dongliu0426** [comment]
oh I didn't see that.

**Comment by maggie-j-liu** [comment]
Is it okay to have my solution and explanation here, or should I just link to the editorial? I think it's more convenient but I don't know. @dolphingarlic @dongliu0426

**Comment by dolphingarlic** [comment]
I think you should just link the editorial, but I don't mind having your explanation there

**Comment by maggie-j-liu** [comment]
Ok, the editorial is linked, is there anything else I need to change?

File: solutions/AC Div Game.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+id: div-game
```
Comment (dolphingarlic) [inline solutions/AC Div Game.mdx:2]:
```suggestion
id: AC-DivGame
```

File: content/4_Gold/Divis.mdx [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const problems = {
 			'Easy',
 			false,
 			['Prime Factorization'],
-			'Prime factorize the given number. Consider each prime in the factorization separately. For each prime, decrement the exponent by 1 the first time, 2 the second time, and so on, until we can no longer continue without repeating a previously used exponent.'
+			'div-game'
```
Comment (dolphingarlic) [inline content/4_Gold/Divis.mdx:20]:
```suggestion
			'AC-DivGame'
```

**Review by dolphingarlic — CHANGES_REQUESTED** [review]
Just change the ID for consistency, and then it should be good to merge

**Pull request merged**